### PR TITLE
RHDEVDOCS-3167 track 1980541: Specify "OpenShift Elasticsearch Op…

### PR DESCRIPTION
**Pull request by rolfedh**
- Purpose: Update the name of the OpenShift Elasticsearch Operator in the OCP 4.6 documentation. This has already been done for the OCP 4.7+ docs.
- Aligned team: Dev Tools
- For branches: enterprise-4.5 and enterprise-4.6 only
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3167
- Direct link to doc preview: https://deploy-preview-34747--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-deploying.html
- SME review: requested @periklis 
- QE reviewed by @kabirbhartiRH  
- Peer reviewed by @JStickler 
- <All reviews complete. Please merge now>

### CODE REVIEW

File: modules/metering-install-verify.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ $ oc --namespace openshift-metering get csv
 [source,terminal,subs="attributes+"]
 ----
 NAME                                           DISPLAY                  VERSION                 REPLACES   PHASE
-elasticsearch-operator.{product-version}.0-202006231303.p0   Elasticsearch Operator   {product-version}.0-202006231303.p0              Succeeded
+elasticsearch-operator.{product-version}.0-202006231303.p0   OpenShift Elasticsearch Operator   {product-version}.0-202006231303.p0              Succeeded
```
Comment (JStickler) [inline modules/metering-install-verify.adoc:31]: Have we verified that this is the actual output?

**Review by JStickler — APPROVED** [review]
Just one comment about making this change in the output in metering-install-verify.adoc because I'm not 100% sure that engineering made that change in the code.

File: modules/metering-install-verify.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/metering-install-verify.adoc:31]: > Have we verified that this is the actual output?

Thanks for checking. I'll verify the output and update this PR accordingly.

Comment (rolfedh) [reply modules/metering-install-verify.adoc:31]: @kabirbhartiRH would you verify this output for me?

Comment (kabirbhartiRH) [reply modules/metering-install-verify.adoc:31]: @rolfedh The actual output is like 

oc get csv -n openshift-logging
NAME                              DISPLAY                            VERSION    REPLACES   PHASE
cluster-logging.5.1.0-78          Cluster Logging                    5.1.0-78              Succeeded
elasticsearch-operator.5.1.0-99   OpenShift Elasticsearch Operator   5.1.0-99              Succeeded

**Review by periklis — APPROVED** [review]
/lgtm

**Pull request merged**